Bottom: 23bb980bab7109cbf88946585ece4ccd062334be
Top:    0bf16915b2dda59d5ef2439658600378e73a6787
Author: scott snyder <sss@karma>
Date:   2020-02-12 11:29:48 -0500

Fix compilation warnings.


---

diff --git a/src/process.cc b/src/process.cc
index a170654..76502b4 100644
--- a/src/process.cc
+++ b/src/process.cc
@@ -660,13 +660,13 @@ void JetVertexRefiner::process() {
 	//check bness
 	vector<const Track*> tracklist = jetVertices[j][0]->getTracks();
 	vector<const Track*> newlist;
-	int hbtr=-1;
+	//int hbtr=-1;
 	double okbness=-1.0;
 	
 	for(unsigned int ntr=0;ntr<tracklist.size();ntr++){
 	  if(tracklist[ntr]->getBNess()>okbness){
 	    okbness = tracklist[ntr]->getBNess();
-	    hbtr=ntr;
+	    //hbtr=ntr;
 	  }
 	  
 	  if(tracklist[ntr]->E()>=1.0 && tracklist[ntr]->getBNess()<_cfg.cutBNess) continue;
